packages/mpv/mpv.conf

```diff
@@ -0,0 +1,10 @@
+# Default configuration for mpv
+
+# Enable OpenSL ES output, since this should work on all Android devices
+ao=opensles
+
+# Increase audio buffer to help with stuttering with bluetooth devices. See #1127.
+audio-buffer=0.5
+
+# Disable Video Output. Termux doesn't support video output (with the exception of "tct").
+vo=null
```

Comment:
Nothing changed. This part doesn't have anything to do with #1127. This is a fix for the error that appears when you load a music file with album art or a video file:
```
$ mpv 01\ -\ Impossible\ Girl\ \#1.mp3
Playing: 01 - Impossible Girl #1.mp3
 (+) Video --vid=1 [P] 'cover' (mjpeg 700x700)
 (+) Audio --aid=1 (mp3 2ch 44100Hz)
File tags:
 Artist: Kim Boekbinder
 Album: The Impossible Girl
 Album_Artist: Kim Boekbinder
 Comment: Visit http://kimboekbinder.bandcamp.com
 Genre: Alternative
 Title: Impossible Girl #1
 Track: 1/0
Error opening/initializing the selected video_out (--vo) device.
Video: no video


Exiting... (Errors when loading file)
```
It's because the default video driver for mpv is `opengl`, which isn't available in Termux. I don't think the default configuration should error. 
There are three possibilities: 
```
$ mpv --vo=help
Available video outputs:
  null           : Null video output
  image          : Write video frames to image files
  tct            : true-color terminals
```
Out of these three `null` seems to be the most sensible one. `image` is only used in very specific circumstances and `tct` requires additional settings (`--quiet`) to function correctly.
It's also why `--ao=opensles` is (and has been) set as default too. The default list in mpv would error otherwise.